### PR TITLE
Updated shortcuts for multi caret editing

### PR DESCRIPTION
**Pull request by DavidJohnWilliams**

### CODE REVIEW

**Comment by PRMerger12** [comment]
@DavidJohnWilliams : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/ide/finding-and-replacing-text.md [hunk 1]
```diff
@@ -116,9 +116,9 @@ Use the following keys and actions for multi-caret selection behaviors:
 |**Ctrl**+**Alt** + double-click|Add a secondary word selection|
 |**Ctrl**+**Alt** + click + drag|Add a secondary selection|
 |**Shift**+**Alt**+**.**|Add the next matching text as a selection|
-|**Ctrl**+**Shift**+**Alt**+**,**|Add all matching text as selections|
+Updated shortcuts |**Shift**+**Alt**+**;**|Add all matching text as selections|
```
Comment (TerryGLee) [inline docs/ide/finding-and-replacing-text.md:119]: @DavidJohnWilliams This is great; thanks ... but, please remove the text that says "Updated shortcuts" - and instead, update the ms.date value in the metadata with the current date. Thanks. :)

**Review by TerryGLee — CHANGES_REQUESTED** [review]

**Comment by PRMerger9** [comment]
@DavidJohnWilliams : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/ide/finding-and-replacing-text.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Find and replace text, and multi-caret selection
-ms.date: 08/14/2018
+ms.date: 17/10/2020
```
Comment (TerryGLee) [inline docs/ide/finding-and-replacing-text.md:3]: @DavidJohnWilliams Apologies to have to ask for another change, but our publishing system relies on a U.S. date in "middle endian" format, i.e. m/d/yyyy, which is then localized to reflect the date format of other countries. Would you please change this to be 10/17/2020 instead?

**Review by TerryGLee — CHANGES_REQUESTED** [review]

**Comment by PRMerger20** [comment]
@DavidJohnWilliams : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by TerryGLee — APPROVED** [review]

**Comment by TerryGLee** [comment]
#sign-off

**Pull request merged**